code/controllers/subsystem/blackbox.dm

```diff
@@ -168,17 +168,18 @@ SUBSYSTEM_DEF(blackbox)
 		var/mob/LA = L.lastattacker
 		laname = sanitizeSQL(LA.real_name)
 		lakey = sanitizeSQL(LA.key)
-	var/sqlgender = sanitizeSQL(L.gender)
 	var/sqlbrute = sanitizeSQL(L.getBruteLoss())
 	var/sqlfire = sanitizeSQL(L.getFireLoss())
 	var/sqlbrain = sanitizeSQL(L.getBrainLoss())
 	var/sqloxy = sanitizeSQL(L.getOxyLoss())
 	var/sqltox = sanitizeSQL(L.getStaminaLoss())
 	var/sqlclone = sanitizeSQL(L.getStaminaLoss())
 	var/sqlstamina = sanitizeSQL(L.getStaminaLoss())
-	var/coord = sanitizeSQL("[L.x], [L.y], [L.z]")
+	var/x_coord = sanitizeSQL(L.x)

```

Comment:
It can be vv'd by admins. It doesn't need to be sanitized unless you don't trust your admins, but it doesn't hurt to.